### PR TITLE
WS: 033 Desktop styling whales page

### PR DESCRIPTION
**Pull request by zuhalkurt**

### CODE REVIEW

File: frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx [hunk 1]
```diff
@@ -63,30 +63,33 @@ export function MeetTheWhalesPage(): JSX.Element {
...
-                </p>
-              </figure>
+              <h5 className="whales__card__common-name">{w.species.name}</h5>
+              <div className="whales__card__grid">
+                <img className="whales__image" src={w.photoUrl} alt={w.name} />
+                <div className="whales__card__info--wrapper">
+                  <p className="whales__card__info--emphasis">
+                    <span className="whales__card__info__title">
+                      Conservation status:
+                    </span>{" "}
```
Comment (TimLeach635) [inline frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx:73]: Prettier sometimes likes to add these annoying little `{" "}` things... it's not your fault at all, but please remove them!
```suggestion
                    </span>
```

File: frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx [hunk 2]
```diff
@@ -63,30 +63,33 @@ export function MeetTheWhalesPage(): JSX.Element {
...
+                    {w.species.endangeredStatus}
+                  </p>
+                  <p className="whales__card__info--emphasis">
+                    <span className="whales__card__info__title">
+                      Description:{" "}
+                    </span>{" "}
```
Comment (TimLeach635) [inline frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx:79]: `{" "}` things

File: frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx [hunk 3]
```diff
@@ -63,30 +63,33 @@ export function MeetTheWhalesPage(): JSX.Element {
...
+                    {w.description}
+                  </p>
+                  <p className="whales__card__info--sponsors">
+                    <span className="whales__card__info__title">
+                      Sponsored by our members:{" "}
```
Comment (TimLeach635) [inline frontend/src/components/meetTheWhales/MeetTheWhalesPage.tsx:84]: `{" "}`
```suggestion
                      Sponsored by our members:
```

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Just a teeny change that isn't your fault!

**Comment by zuhalkurt** [comment]
Changes made according to comments :)

**Pull request merged**